frontend/src/metabase/admin/people/components/GroupsListing.jsx

```diff
@@ -218,6 +277,12 @@ function GroupsTable({
   onEditGroupCancelClicked,
   onEditGroupDoneClicked,
 }) {
+  const [apiKeys, setApiKeys] = useState([]);
+
+  useEffect(() => {
+    ApiKeysApi.list().then(setApiKeys);
+  }, []);
```

Comment:
Same as above, though this one will actually change which modals pop up because that's determined by state which may not have been set yet. Showing a loading spinner until all data is acquired is probably the right thing to do here